config.toml

```diff
@@ -88,12 +90,27 @@ paginate = 6
     # name = "License and Funding"
     # URL = "/privacy"
     # weight = 20
+
+  [[menu.main]]
+  weight = 9
+  name = "Examples"
+  URL = "/examples"
   
 
   [[menu.main]]
-  name = "QCA Infrastructure"
-  URL = "/projects"
+  name = "Infrastructure"
+  URL = "https://docs.qcarchive.molssi.org"
```

Comment:
This will be the landing page for the infrastructure? Out of curiosity, what repo will it live in?

Reply:
One more repo ;/

https://github.com/MolSSI/QCArchiveInfrastructureWebsite